src/test/kotlin/blackjack/domain/player/DealerTest.kt

```diff
@@ -0,0 +1,97 @@
+package blackjack.domain.player
+
+import blackjack.domain.card.Ace
+import blackjack.domain.card.CardType
+import blackjack.domain.card.Five
+import blackjack.domain.card.Four
+import blackjack.domain.card.Jack
+import blackjack.domain.card.King
+import blackjack.domain.card.Nine
+import blackjack.domain.card.Queen
+import blackjack.domain.card.Seven
+import blackjack.domain.card.Six
+import blackjack.domain.card.Two
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+/**
+ * Created by Jaesungchi on 2022.06.18..
+ */
+
+class DealerTest {
+    @Test
+    fun `딜러가 15 유저가 17인 경우 딜러가 유저의 배팅금액만큼 잃는다`() {
+        val cards = listOf(Four(CardType.DIAMOND), Ace(CardType.CLUB))
+        val dealer = Dealer(cards)
```

Comment:
```suggestion
        val dealer = Dealer(Four(DIAMOND), Ace(CLUB))
```
저는 종종 테스트 가독성을 위해 일급 컬렉션의 경우 가변인자를 받는 부 생성자를 추가하기도 합니다. static import 도 적극 활용하고요.

Reply:
아하 좋은 정보 감사합니다 ㅎㅎ 
현재 제가 일하는 팀의 컨벤션에서는 static import 사용시 코드를 볼때 이게 이 클래스 내부에 있는 건지, 외부에서 static으로 가져온건지 구분하기가 힘들어서, static 메소드나 값을 들고오는 경우에는 앞에 붙이기로 했었는데, 익숙해서 이렇게 쓰게 되었네요 ㅎㅎ;